Formula/k/kubevpn.rb

```diff
@@ -0,0 +1,25 @@
+class Kubevpn < Formula
+  desc "Offers a Cloud-Native Dev Environment that connects to your K8s cluster network"
+  homepage "https://www.kubevpn.cn"
+  url "https://github.com/kubenetworks/kubevpn.git",
+      tag:      "v2.2.11",
+      revision: "98d88ac542c06e8f66937e7f57b5ceae514b0c30"
```

Comment:
let's use github tarball file

Reply:
but i found if use tarball file, can't get commitid, we needs commitid to compare match or not

Reply:
yeah, we dont need commits at all (the commit is good for debugging build, not release builds)

Reply:
but kubevpn needs commitid to compare version, to update k8s deploy image tag.

Reply:
@chenrui333 can we just keep in this way?

Reply:
> but kubevpn needs commitid to compare version, to update k8s deploy image tag.

can you point me how that is done? I would be interested about it, thanks!

Reply:
> but kubevpn needs commitid to compare version, to update k8s deploy image tag.



> > but kubevpn needs commitid to compare version, to update k8s deploy image tag.
> 
> can you point me how that is done? I would be interested about it, thanks!

ok, but to be clear, i just use tag to upgrade deploy pod image tag.

Reply:
yeah, homebrew-core does not autoupdate. so it is better to print a warning when the version mismatch.

in this case, I can see you can totally just use version rather than commitId (also since we have checksum for version, it is immutable)

Reply:
how about just keep in this way. i see homebrew-core repo k9s also use this way. i don't know why i can't use commitid

Reply:
yeah, just use version.

Reply:
we `must` use github tarball file? not `recommend`？but homebrew-core privide 
```yaml
  url "https://github.com/kubenetworks/kubevpn.git",
      tag:      "v2.2.11",
      revision: "98d88ac542c06e8f66937e7f57b5ceae514b0c30"
```
why we can't use this way ? 

i see k9s.rb also use this way. can you explain why?
```yaml
class K9s < Formula
  desc "Kubernetes CLI To Manage Your Clusters In Style!"
  homepage "https://k9scli.io/"
  url "https://github.com/derailed/k9s.git",
      tag:      "v0.32.5",
      revision: "1440643e8d1a101a38d9be1933131ddf5c863940"
  license "Apache-2.0"
```

Reply:
@chenrui333 i can not found any doc or reference to indicate that. DO NOT USE COMMITID. i follow the cook book to commit new formula.

Reply:
@chenrui333 if homebrew do really have this limit or requirement. Of course i can change the source code. But i can not found any doc

Reply:
@chenrui333 forget it. 

can you help to delete this file, it makes CI failed

```shell
/home/linuxbrew/.cache/Homebrew/downloads/559921f4d97e89e35c4a33203d9459ddf87911cc5e00f319dd6cdcedeed24abf--kubevpn-2.2.11.tar.gz
  To retry an incomplete download, remove the file above.
```

Reply:
here is commit https://github.com/kubenetworks/kubevpn/commit/d6f833fc0b3246e152d3d952f8f86ef2ec7fe68b